docs/assemblies/scaling.adoc

```diff
@@ -94,6 +94,26 @@ $ oc rsh openstackclient openstack hypervisor list
 +--------------------------------------+-------------------------------------+-----------------+-----------------+-------+
 ----
 
+== Scaling Out with different configuration
+
+If the deployment needs to be scaled out to nodes that require different
+configuration (e.g. different kernel args, network config, or openstack config)
+compared to the configuration in the current `OpenStackDataPlaneNodeSet`
+then the new nodes cannot be added to the existing `OpenStackDataPlaneNodeSet`
+but a new `OpenStackDataPlaneNodeSet` needs to be created that contains the
+new nodes and the new configuration for those nodes. Then a new
+`OpenStackDataPlaneDeployment` needs to be created that points to *both the
+existing and the new `OpenStackDataPlaneNodeSets`* to trigger the scale out.
+
+WARNING: The existing `OpenStackDataPlaneNodeSet` has the `ssh-known-hosts`
+service. That service is global for the whole deployment so it should not be
+listed on the new `OpenStackDataPlaneNodeSet`.
+
+WARNING: If only the new `OpenStackDataPlaneNodeSet` is included into the new
+`OpenStackDataPlaneDeployment` then the scale out seems to succeed but will
+be incomplete causing that VM move operations will fail between nodes
+in the different `OpenStackDataPlaneNodeSets`.
+
 == Scaling In
```

Comment:
we proably need ot also update the scale in procedure for the removal of node sets
to as we should also removed those form the know howst file eventully.

Reply:
Yeah I can add a note to scale in if you removed the last node from a nodeset then please delete the nodeset, and if this was the nodeset listing the ssh-known-hosts service then add that service to another not deleted nodeset.

Reply:
done